pkg/params/settings/config.go

```diff
@@ -40,26 +40,28 @@ type HubCatalog struct {
 	URL  string
 }
 
+// if there is a change performed on the default value,
+// update the same on "config/302-pac-configmap.yaml"
 type Settings struct {
 	ApplicationName                    string `default:"Pipelines as Code CI" json:"application-name"`
 	HubCatalogs                        *sync.Map
-	RemoteTasks                        bool   `default:"true"                                json:"remote-tasks"`
+	RemoteTasks                        bool   `default:"true" json:"remote-tasks"`
 	MaxKeepRunsUpperLimit              int    `json:"max-keep-run-upper-limit"`
 	DefaultMaxKeepRuns                 int    `json:"default-max-keep-runs"`
-	BitbucketCloudCheckSourceIP        bool   `default:"true"                                json:"bitbucket-cloud-check-source-ip"`
+	BitbucketCloudCheckSourceIP        bool   `default:"true" json:"bitbucket-cloud-check-source-ip"`
 	BitbucketCloudAdditionalSourceIP   string `json:"bitbucket-cloud-additional-source-ip"`
 	TektonDashboardURL                 string `json:"tekton-dashboard-url"`
-	AutoConfigureNewGitHubRepo         bool   `default:"false"                               json:"auto-configure-new-github-repo"`
+	AutoConfigureNewGitHubRepo         bool   `default:"false" json:"auto-configure-new-github-repo"`
 	AutoConfigureRepoNamespaceTemplate string `json:"auto-configure-repo-namespace-template"`
 
-	SecretAutoCreation               bool   `default:"true"                             json:"secret-auto-create"`
-	SecretGHAppRepoScoped            bool   `default:"true"                             json:"secret-github-app-token-scoped"`
+	SecretAutoCreation               bool   `default:"true" json:"secret-auto-create"`
+	SecretGHAppRepoScoped            bool   `default:"true" json:"secret-github-app-token-scoped"`
 	SecretGhAppTokenScopedExtraRepos string `json:"secret-github-app-scope-extra-repos"`
 
-	ErrorLogSnippet             bool   `default:"true"                                                                          json:"error-log-snippet"`
-	ErrorDetection              bool   `default:"true"                                                                          json:"error-detection-from-container-logs"`
-	ErrorDetectionNumberOfLines int    `default:"50"                                                                            json:"error-detection-max-number-of-lines"`
-	ErrorDetectionSimpleRegexp  string `default:"^(?P<filename>[^:]*):(?P<line>[0-9]+):(?P<column>[0-9]+):([ ]*)?(?P<error>.*)" json:"error-detection-simple-regexp"`
+	ErrorLogSnippet             bool   `default:"true" json:"error-log-snippet"`
+	ErrorDetection              bool   `default:"true" json:"error-detection-from-container-logs"`
+	ErrorDetectionNumberOfLines int    `default:"50" json:"error-detection-max-number-of-lines"`
+	ErrorDetectionSimpleRegexp  string `default:"^(?P<filename>[^:]*):(?P<line>[0-9]+):(?P<column>[0-9]+)?([ ]*)?(?P<error>.*)" json:"error-detection-simple-regexp"`
```

Comment:
why the change between `?` and `:`

Reply:
@chmouel I see already you have updated the config map with `?` but not updated here
https://github.com/openshift-pipelines/pipelines-as-code/pull/1232

Reply:
i see yeah, we need to figure out a way to have a CI job to sync or check the two because this becomes a pain